dokka-runners/dokka-gradle-plugin/src/main/kotlin/internal/PluginFeaturesService.kt

```diff
@@ -112,6 +118,39 @@ internal abstract class PluginFeaturesService : BuildService<PluginFeaturesServi
         }
     }
 
+    internal val enableK2Analysis: Boolean by lazy(SYNCHRONIZED) {
+        // use lazy {} to ensure messages are only logged once
+
+        val enableK2Analysis = parameters.k2AnalysisEnabled.getOrElse(false)
+
+        if (enableK2Analysis) {
+            logK2AnalysisMessage()
+        }
+
+        enableK2Analysis
+    }
+
+    private fun logK2AnalysisMessage() {
+        if (primaryService && !parameters.k2AnalysisNoWarn.getOrElse(false)) {
+            logger.warn(
+                """
+                |Dokka K2 Analysis is enabled
+                |
+                |  This feature is Experimental and is still under active development.
```

Comment:
Curious - why 2 space indent for the major part of the message? :thinking:

Reply:
I have no strong reason. My intention was to differentiate between the title and body of the message, to make the message easier to scan quickly.